app/src/screens/MapScreen.js

```diff
@@ -393,6 +421,17 @@ const useStyles = makeStyles(theme => ({
   title: {
     flexGrow: 1,
   },
+  sideBar: {
+    padding: 10,
+    background: 'rgba(255,255,255,0.8)',
+    display: 'inline-block',
+    'max-height': '90%',
+    'overflow-y': 'auto',
+    'overflow-x': 'hidden',
+  },
```

Comment:
it looks like this is supposed to add a scrollbar, but when I tried it out I couldn't find the scrollbar/figure out how to use it when the sidebar was expanded past the laptop screen :/

Reply:
Yea... you have to click on the actual scrollbar, otherwise it doesn't work.
I have no idea how to get around this as it seems the map steals the mouse input.

I figured scroll bar is better than nothing. Hopefully @apbuteau's UI changes make everything better placed so this isn't an issue.